plugins/vdb/configs/config.d/10_vdb.json

```diff
@@ -0,0 +1,8 @@
+{
+  ".*(vdb)":
+  {
+    "load-plugins": "vdb",
+    "volume": true,
+    "inverse": true
```

Comment:
any reason `inverse` is true by default?

Reply:
Because this is what the majority of the dataset available in openvdb.com/download needs. But it is not absolute. a .vdb file can be not inversed and can also not contain any volume but a point cloud.